scipy/misc/common.py

```diff
@@ -201,3 +202,70 @@ def face(gray=False):
     if gray is True:
         face = (0.21 * face[:,:,0] + 0.71 * face[:,:,1] + 0.07 * face[:,:,2]).astype('uint8')
     return face
+
+
+def electrocardiogram(physical_unit=True):
+    """
+    Load an electrocardiogram as an example for a one-dimensional signal.
+
+    The returned signal is a 5 minute long electrocardiogram (ECG), a medical
+    recording of the hearts electrical activity, sampled at 360 Hz.
+
+    Parameters
+    ----------
+    physical_unit : bool, optional
+        If true, the returned ECG uses the physical unit millivolt (mV)
+        otherwise the raw output of the ADC is returned.
```

Comment:
I don't think we really need this. Let's always return `mV`. If someone really wants raw ADC values they can go to the original data source.